test/spec_helper.rb

```diff
@@ -54,7 +54,7 @@ def reset_caches
         dummy_config.assets.version = version
         dummy_assets.version = version
         cache = dummy_assets.cache
-        cache.respond_to?(:clear) ? cache.clear : rm_r("#{dummy_tmp}/cache/assets")
+        cache.respond_to?(:clear) ? cache.clear : rm_rf("#{dummy_tmp}/cache/assets")
```

Comment:
@simi Because some tests failed here at Rails 4.2 like:
```
  1) Error:
config#test_0003_must have an options hash passed down to the #to_css method:
Errno::ENOENT: No such file or directory @ unlink_internal - /home/travis/build/metaskills/less-rails/test/dummy_app/tmp/cache/assets
    /home/travis/.rvm/rubies/ruby-2.1.4/lib/ruby/2.1.0/fileutils.rb:1460:in `unlink'
    /home/travis/.rvm/rubies/ruby-2.1.4/lib/ruby/2.1.0/fileutils.rb:1460:in `block in remove_file'
    /home/travis/.rvm/rubies/ruby-2.1.4/lib/ruby/2.1.0/fileutils.rb:1465:in `platform_support'
    /home/travis/.rvm/rubies/ruby-2.1.4/lib/ruby/2.1.0/fileutils.rb:1459:in `remove_file'
    /home/travis/.rvm/rubies/ruby-2.1.4/lib/ruby/2.1.0/fileutils.rb:1448:in `remove'
    /home/travis/.rvm/rubies/ruby-2.1.4/lib/ruby/2.1.0/fileutils.rb:776:in `block in remove_entry'
    /home/travis/.rvm/rubies/ruby-2.1.4/lib/ruby/2.1.0/fileutils.rb:1502:in `postorder_traverse'
    /home/travis/.rvm/rubies/ruby-2.1.4/lib/ruby/2.1.0/fileutils.rb:774:in `remove_entry'
    /home/travis/.rvm/rubies/ruby-2.1.4/lib/ruby/2.1.0/fileutils.rb:632:in `block in rm_r'
    /home/travis/.rvm/rubies/ruby-2.1.4/lib/ruby/2.1.0/fileutils.rb:628:in `each'
    /home/travis/.rvm/rubies/ruby-2.1.4/lib/ruby/2.1.0/fileutils.rb:628:in `rm_r'
    /home/travis/build/metaskills/less-rails/test/spec_helper.rb:57:in `reset_caches'
    /home/travis/build/metaskills/less-rails/test/cases/railtie_spec.rb:19:in `block (2 levels) in <class:RailtieSpec>'
```
I don't know why  but I changed `rm_r` to `rm_rf` ,  all tests passed.